docs/reference/extensions/go-framework.rst

```diff
@@ -0,0 +1,71 @@
+.. _go-framework-reference:
+
+go-framework
+----------------
+
+The Go extension streamlines the process of building Go application
+rocks.
+
+The extension builds and copies the Go binary file to the rock,
+inside the ``/app`` directory. By default, the base ``bare`` is used,
```

Comment:
In the final image, it looks like the binary is in `/usr/local/bin` and `/app` is empty though.

```
$ docker run hello:0.1 ls /app
$ docker run hello:0.1 ls /usr/local/bin
hello
```